IFC 4.3/SpanAnnotation_1/UT_SpanAnnotation_1.ifc

```diff
@@ -41,7 +41,7 @@ DATA;
 #24= IFCAXIS2PLACEMENT3D(#23,$,$);
 #25= IFCDIRECTION((0.0,1.0));
 #26= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body','Model',*,*,*,*,#22,$,.MODEL_VIEW.,$);
-#27= IFCSHAPEREPRESENTATION(#26,'Body','Curve',(#21));
+#27= IFCSHAPEREPRESENTATION(#26,'Axis2D','Curve',(#21));
```

Comment:
When reading the documentation on IfcShapeRepresentation, I would have guessed RepresentationIdentifier='Axis' and RepresentationType='Curve2D', i.e. '2D' is in the wrong place?

Reply:
Sorry for not being clear enough. What @larswik said is correct.

Reply:
see c0bb0e8